sevntu-checks/pom.xml

```diff
@@ -25,7 +25,7 @@
     <maven.jacoco.plugin.version>0.8.7</maven.jacoco.plugin.version>
     <maven.pmd.plugin.version>3.11.0</maven.pmd.plugin.version>
     <pmd.version>6.38.0</pmd.version>
-    <java.version>1.8</java.version>
+    <java.version>11</java.version>
```

Comment:
When this is ready for merge, can you provide an example of using the Checkstyle JDK11 All JAR with Sevntu's JDK11 JAR and show a custom checks in Sevntu will work from the CLI using JDK11. I want  to verify an integration test to show sevntu works and we are not fooling ourselves with anything in our testing area.

I think manually testing will be enough, but I am not against from building a CI item to show this.

Reply:
```
rivanov@p5510:/var/tmp$ cat Test.java 
public class Test
{
    private final                                                                                                     MyThing thing;

}
rivanov@p5510:/var/tmp$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
        <module name="LineLengthExtended"/>
    </module>
</module>
rivanov@p5510:/var/tmp$ java -classpath sevntu-checks-1.41.0.jar:checkstyle-10.2-all.jar com.puppycrawl.tools.checkstyle.Main -c config.xml Test.java 
Starting audit...
[ERROR] /var/tmp/Test.java:3: Line is longer than 80 characters. Current line length is 132. [LineLengthExtended]
Audit done.
Checkstyle ends with 1 errors.
```

sevntu version looks old, but it is how it working now (no snapshot, new version appears during release, but I took this jar from target folder)